.travis.yml

```diff
@@ -7,7 +7,7 @@ scala:
 - 2.13.0
 
 jdk:
-- openjdk8
+- openjdk11
```

Comment:
Is there a reason why not worth adding instead of replacing?

Reply:
Well, let's say that the reason is Travis. It looks like there is a bit of a mess around the Jdk versions supported by `bionic`. What I mean is [this](https://travis-ci.community/t/openjdk8-on-bionic/4373) and this https://github.com/travis-ci/docs-travis-ci-com/pull/2444 .
We are not in a rush on this change so we can wait, and if they decide that they will not support `openjdk8` we might think again if we want to add a customization for that to the build. Or we might even consider migrating to Circle CI.